server/src/test/java/io/mantisrx/master/jobcluster/job/JobClusterManagerTest.java

```diff
@@ -420,12 +420,7 @@ public void testBootStrapJobClustersAndJobs() {
                         "host1",
                         "vm1",
                         empty(),
-                        new WorkerPorts(
-                                8000,
-                                9000,
-                                9010,
-                                9020,
-                                Lists.newArrayList(9030)));
+                        new WorkerPorts(Lists.newArrayList(8000, 9000, 9010, 9020, 9030)));
```

Comment:
For this test, and other tests, they were initially failing. This was because the tests use a different constructor than `SchedulingService`. We can update the `WorkerPort` constructor(s), but they aren't being used. It's better to make the usage match what the scheduler is really using.

Reply:
@neerajrj yeah we've got invalid port tests from https://github.com/Netflix/mantis/blob/master/mantis-common/src/test/java/io/mantisrx/common/WorkerPortsTest.java. Also for the master restart: https://github.com/Netflix/mantis-control-plane/blob/master/server/src/test/java/io/mantisrx/master/jobcluster/job/JobClusterManagerTest.java#L624.